common/common.go

```diff
@@ -135,3 +136,45 @@ func SupportsOverlay() bool {
 	}
 	return false
 }
+
+// PrivateNetList implements the flag.Value interface to allow specification
+// of -private-net with and without values
+type PrivateNetList struct {
+	mapping map[string]bool
```

Comment:
I meant like this: http://play.golang.org/p/Z6Id-ESO3p

But now looking at this, `bool` seems cleaner. Or put it another way, both look bad -- Go needs a `set` type.